drivers/can/can_nrf.c

```diff
@@ -177,6 +181,14 @@ static int can_nrf_init(const struct device *dev)
 		return ret;
 	}
 
+
+#ifdef CONFIG_SOC_NRF54H20_GPD
+	ret = nrf_gpd_retain_pins_set(config->pcfg, false);
```

Comment:
I think this should be done after `TASKS_START` to avoid glitches on pins. I guess CAN starts controlling the pins after TASKS_START is called

Reply:
done